specification/semantic_conventions.md

```diff
@@ -129,6 +129,9 @@ instances. For each `LogRecord` instance:
   - `pprof-gzip-base64`,
   - `text` ([Deprecated](../README.md#versioning-and-status-of-the-specification)
     format).
+- `profiling.instrumentation.source` MUST be set to either:
+  - `continuous` for continuous "Always On" profiling
+  - `snapshot` for trace snapshot profiling
```

Comment:
I think that, for backward compatibility, you should mention that it will be treated as `continuous`.

Reply:
This, there are too many existing installations, my suggestion is that `profiling.instrumentation.source` must only be set when anything other than `continuous` is used (as the backend treats all of the profiles as continuous by default).